src/main/resources/crafter/studio/upgrade/pipelines.yaml

```diff
@@ -404,6 +404,14 @@ pipelines:
               dest: /configuration/samples/sample-webdav.xml
           commitDetails: Update sample files
         - type: dbVersionUpgrader
+    - currentVersion: 3.1.0.32
+      nextVersion: 3.1.0.33
+      operations:
+        - type: xsltFileUpgrader
+          path: /configuration/samples/sample-notification-config.xml
+          template: crafter/studio/upgrade/3.1.x/3.1.0.33/system/sample-notification-config.xslt
+          commitDetails: Add pull from remote conflict notification
+        - type: dbVersionUpgrader
```

Comment:
We have been forgetting to update the version in the create db script for a while, please add it